src/matrix/room/Invite.js

```diff
@@ -58,7 +58,7 @@ export class Invite extends EventEmitter {
 
     /** @see BaseRoom.avatarColorId */
     get avatarColorId() {
-        return this._inviteData.avatarColorId;
+        return this._inviteData.avatarColorId || this.id;
```

Comment:
Invites are only process once, when they are received, so if a client inserted `null` into storage, that `null` stays there. This line could be used to mitigate the issue.